Tests/scripts/infrastructure_tests/update_id_set_test.py

```diff
@@ -161,5 +162,33 @@ def test_get_playbook_data(self):
         self.assertDictEqual(data['command_to_integration'], PLAYBOOK_DATA['command_to_integration'])
 
 
+PLAYBOOK_DATA2 = PLAYBOOK_DATA.copy()
+PLAYBOOK_DATA2["implementing_playbooks"] = ["Cortex XDR Incident Handling"]
+PLAYBOOK_DATA2["name"] = "Calculate Severity - Standard"
+
+PLAYBOOK_DATA3 = PLAYBOOK_DATA.copy()
+PLAYBOOK_DATA3["implementing_playbooks"] = ["Cortex XDR Incident Handling"]
+PLAYBOOK_DATA3["name"] = "Palo Alto Networks - Malware Remediation"
+
+playbook_deps = [
+    ({"playbooks": [{"str": PLAYBOOK_DATA}]}, True),
+    ({"playbooks": [{"str0": PLAYBOOK_DATA}, {"str1": PLAYBOOK_DATA2}, {"str2": PLAYBOOK_DATA3}]},
+     False),
+    ({}, False),
+    ({"playbooks": [{"str": PLAYBOOK_DATA}],
+      "TestPlaybooks": [{"str": PLAYBOOK_DATA2}, {"str1": PLAYBOOK_DATA3}]}, False),
+    ({"playbooks": [{"str": PLAYBOOK_DATA}], "TestPlaybooks": [{"str": PLAYBOOK_DATA2}]}, True)
+]
+
+
+@pytest.mark.parametrize('input_data, expected_error', playbook_deps)
+def test_validate_playbook_deps(input_data, expected_error):
```

Comment:
this test is not clear at all. please rewrite it, for each behaviour write a separate unit test. in unit test explain the behaviour/use case.
What was in the system? what was the trigger? then what you expect to happen?

scenario: playbook depend on sub playbook that don't exists
given
- playbook A depend on sub playbook B

when
- running validate

then
- validate should raise an KeyError exception ...

Like that